rc/lsp.kak

```diff
@@ -1415,6 +1417,26 @@
 " | eval "${kak_opt_lsp_cmd} --request") > /dev/null 2>&1 < /dev/null & }
 }
 
+define-command -hidden lsp-breadcrumbs-request -docstring "request updating lsp_modeline_breadcrumbs for the buffer" %{
```

Comment:
I'd avoid implementation details (variable name) in comments. Also it should be window scope

```suggestion
define-command -hidden lsp-breadcrumbs-request -docstring "request updating modeline breadcrumbs for the window" %{
```